### PR TITLE
nodeSelector-api-fix: updates apiVersion for nodeSelector

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s): 4.13+

Issue:

Link to docs preview:
https://62462--docspreview.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/configuring-egress-firewall-ovn.html#configuringNodeSelector-example_configuring-egress-firewall-ovn

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jab-rh** [comment]
@JoeAldinger, LGTM

File: modules/nw-egressnetworkpolicy-object.adoc [hunk 1]
```diff
@@ -151,9 +151,9 @@ spec:
 
 As a cluster administrator, you can allow or deny egress traffic to nodes in your cluster by specifying a label using `nodeSelector`. Labels can be applied to one or more nodes. The following is an example with the `region=east` label:
 
-[source,yaml]
+[source,yaml,subs="attributes+"]
 ----
-apiVersion: v1
+apiVersion: {api}
```
Comment (huiran0826) [inline modules/nw-egressnetworkpolicy-object.adoc:156]: What's the value of {api}?

Comment (JoeAldinger) [reply modules/nw-egressnetworkpolicy-object.adoc:156]: Sorry, it renders as `apiVersion: k8s.ovn.org/v1` in the preview.

**Review by adellape — APPROVED** [review]

**Pull request merged**